csharp/Ice/CustomError/Server/Chatbot.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) ZeroC, Inc.
+
+using VisitorCenter;
+
+namespace CustomErrorServer;
+
+/// <summary>Chatbot is an Ice servant that implements Slice interface Greeter.</summary>
+internal class Chatbot : GreeterDisp_
+{
+    private const int MaxLength = 7;
+
+    /// <inheritdoc/>
+    // Implements the abstract method Greet from the GreeterDisp_ class generated by the Slice compiler.
+    // This variant is the synchronous implementation.
+    public override string Greet(string name, Ice.Current current)
+    {
+        Console.WriteLine($"Dispatching greet request {{ name = '{name}' }}");
+
+        if (name.Length > MaxLength)
```

Comment:
You should use a switch expression, just like:
https://github.com/icerpc/icerpc-csharp/blob/f109b98c1cb69755ccca7ddf3cc247982e7afb8c/examples/slice/CustomError/Server/Chatbot.cs#L23

```
return name switch
{
   ...
}
```

and you can throw after `=>`.